src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -32,7 +33,7 @@ public void testHanoiRhymerReportRejected() {
     @Test
     public void testHanoiRhymerCountIn() {
         HanoiRhymer rhymer = new HanoiRhymer();
-        final int EMPTY_STACK_VALUE = -1;
+        final int EMPTY_STACK_VALUE = 0;
```

Comment:
original constant should be used (it would be best to use a common constant in a containers interface).